collect_app/src/test/java/org/odk/collect/android/application/RobolectricApplication.java

```diff
@@ -18,6 +20,9 @@
 
     @Override
     public void onCreate() {
+        // Make sure storage is accessible
+        ShadowEnvironment.setExternalStorageState(MEDIA_MOUNTED);
```

Comment:
We don't actually need this for the final implementation, but it should avoid problems in the future with objects that check this.